From: Gordon Malm <bugs-gentoo-org-02@bumpin.org>

ia64: Fix kernel compile failure with loadable module support enabled.

This patch is present in upstream grsecurity patches as of
pax-linux-2.6.24.1-test12.patch. This patch can be dropped for any
hardened-sources-2.6.24 based upon pax-linux-2.6.24.1-test12.patch or
later.

Acked-by: Kerin Millar <kerframil@gmail.com>

--- a/arch/ia64/kernel/module.c
+++ b/arch/ia64/kernel/module.c
@@ -531,7 +531,7 @@ in_core_rw (const struct module *mod, ui
 static inline int
 in_core (const struct module *mod, uint64_t addr)
 {
-	return in_core_rx(mod, value) || in_core_rw(mod, value);
+	return in_core_rx(mod, addr) || in_core_rw(mod, addr);
 }
 
 static inline int
